src/plugins/kibana_usage_collection/server/collectors/application_usage/schema.ts

```diff
@@ -152,7 +152,6 @@ export const applicationUsageSchema = {
   fleet: commonSchema,
   integrations: commonSchema,
   ingestManager: commonSchema,
-  investigate: commonSchema,
```

Comment:
We'll need `investigate` when the feature flag is true. Are the telemetry functional tests green locally?

Reply:
nevermind